activemq_xml/datadog_checks/activemq_xml/data/conf.yaml.example

```diff
@@ -9,38 +35,177 @@ instances:
     #
   - url: http://localhost:8161
 
+  ## @param suppress_errors - boolean - optional - default: false
```

Comment:
`(L38) Content is not correctly indented - skipping rest of file`

two more spaces for the next 25 lines :)

Reply:
ty!